libbeat/outputs/elasticsearch/client_integration_test.go

```diff
@@ -54,7 +54,11 @@ func TestClientPublishEvent(t *testing.T) {
 	testPublishEvent(t, index, cfg)
 }
 
+// this test is broken 21 fails of 21 builds is not flaky at all
+// https://github.com/elastic/beats/issues/21295
+// https://github.com/elastic/beats/pull/21296
```

Comment:
Why not approving the original PR? :slightly_smiling_face: https://github.com/elastic/beats/pull/21296

Reply:
here is informative in order to have all the test, this change can be removed if we merge the other PR